app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -538,8 +538,8 @@ class _CustomChatInputState extends ConsumerState<CustomChatInput> {
 
   Future<void> handleFileUpload(List<File> files) async {
     final roomId = widget.convo.getRoomIdStr();
-    final chatInputState = ref.read(chatInputProvider(roomId));
-    final chatInputNotifier = ref.read(chatInputProvider(roomId).notifier);
+    final client = ref.read(clientProvider)!;
```

Comment:
please also add null handler for it too
```suggestion
    final client = ref.read(clientProvider);
    if(client == null){
    throw Unimplemented or Unavailable error; 
    }
```

Reply:
Will fix it in [another PR](https://github.com/acterglobal/a3/pull/1214), because there are many occurrences of client provider on flutter side.